substrate/client/rpc-spec-v2/src/transaction/tests/transaction_broadcast_tests.rs

```diff
@@ -26,6 +26,8 @@ use std::sync::Arc;
 use substrate_test_runtime_client::AccountKeyring::*;
 use substrate_test_runtime_transaction_pool::uxt;
 
+const MAX_TX_PER_CONNECTION: usize = 4;
```

Comment:
As a nit, the spec only says "at least 4 transactions can be broadcast at the same time". Do we have any reason to also use 4 as the upper bound here?

(One reason I can think of is just that it nudges clients to comply with that bit of the spec, ie if they try to send more than 4, which _might_ not be supported per the spec, then an error will come back)